kdumpctl

```diff
@@ -962,8 +962,22 @@ check_dump_feasibility()
 	check_kdump_feasibility
 }
 
+fadump_bootargs_append()
+{
+	if [[ -f "$FADUMP_APPEND_ARGS_SYS_NODE" ]]; then
+		output=$( { echo "${FADUMP_COMMANDLINE_APPEND}" > "$FADUMP_APPEND_ARGS_SYS_NODE" ; } 2>&1)
```

Comment:
Hi Hari,
just a small nit. Could you please define `output` as `local`. Other than that the commit is fine.
Thanks
Philipp